AWS_LAMBDA/remove_predictions_from_exited_guests/lambda_function.py

```diff
@@ -0,0 +1,69 @@
+# Each day, guests are exiting. Once a guest has exited, the predicted_exit_destination
+# column should be "exited" rather than a prediction.
+
+import os
+import sys
+import logging
+import psycopg2
+
+# RDS settings
+rds_host = os.environ.get('RDS_HOST')
+rds_username = os.environ.get('RDS_USERNAME')
+rds_user_pwd = os.environ.get('RDS_USER_PWD')
+
+# Logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Connect to database
+try:
+    # If able to connect
+    conn = psycopg2.connect(
+        host=rds_host,
+        user=rds_username,
+        password=rds_user_pwd)
+except:
+    # If unable to connect
+    logger.error("ERROR: Could not connect to Postgres instance.")
+    sys.exit()
+
+# Log connection success
+logger.info("SUCCESS: Connection to RDS Postgres instance succeeded")
+
+
+def lambda_handler(event, context):
+    '''Add predicted_exit_destination column to guests_temp if not exists, 
+    then updates guests_temp to account for newly exited guests (if exited, no more need for prediction on row).'''
+
+    # Check if column exists
+    if conn != None: 
```

Comment:
I don't understand why this is necessary if `conn` is already accounted for in a try block on line 19, or in other words, `conn` should never be `None` or else it would except on line 25 and never reach this line.

Maybe we move line 56 up here and add the remaining logic after line 56 inside a `finally` block along with commit/close.

Reply:
Originally had line 56 (with conn.cursor() as cur) at this point. But it will not work with a python try and except if both require executing a query.

Reply:
However I can remove the if statement. That was more me being overly cautious.